core/src/main/java/org/elasticsearch/index/query/TermsQueryBuilder.java

```diff
@@ -158,76 +287,127 @@ public TermsQueryBuilder disableCoord(boolean disableCoord) {
         return this;
     }
 
+    public boolean disableCoord() {
+        return this.disableCoord;
+    }
+
+    public boolean isTermsLookupQuery() {
+        return this.termsLookup.id != null;
```

Comment:
what if termsLookup is null? can't we just do `return this.termsLookup != null` ?